### PR TITLE
Global variables new syntax

### LINKED ISSUE #125: Update Global Variables Syntax
Global variables should follow this syntax $global--concept--PropiertyCamelCase--modifier--state

### PR DESCRIPTION
**Pull request by andresgalante**
Global variables now follow this syntax `$global--concept--PropiertyCamelCase--modifier--state`

This is a global change in every file to reflect that.

This PR will destroy #68 but I think we should merge this and do the turn variables from CSS to SASS before fixing it.

resolves #125

### CODE REVIEW

File: src/patternfly/components/Content/styles.scss [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 .pf-c-content {
   a {
-    color: $pf-global--link--Color;
-    text-decoration: $pf-global--link--TextDecoration;
+    color: $global--link--Color;
+    text-decoration: $global--link--TextDecoration;
```
Comment (cliffpyles) [inline src/patternfly/components/Content/styles.scss]: Are these suppose to be custom properties or sass variables in this PR? or does it matter?

File: src/patternfly/components/Content/styles.scss [hunk 2]
```diff
@@ -40,70 +40,70 @@
...
-    font-size: $pf-global--caption--FontSize;
+    margin-bottom: $global--spacer--lg;
+    font-size: $global--FontSize--sm;
   }
 
   blockquote {
-    padding: $pf-global--spacer--md;
-    border-left: 5px solid $pf-global-dark-off-color;
+    padding: $global--spacer--lg;
+    border-left: 5px solid $global--Color--dark-alt;
```
Comment (cliffpyles) [inline src/patternfly/components/Content/styles.scss]: I think this one was probably meant to be something else. It's using a variable that is only meant to be used with the color property.

File: src/patternfly/patterns/SecondaryNav/styles.scss [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-$pf-p-secondary-nav__link--PaddingTop:           $pf-global--spacer--sm !default;
-$pf-p-secondary-nav__link--PaddingBottom:        $pf-global--spacer--sm !default;
+$pf-p-secondary-nav__link--Color:                $global--Color--dark !default;
+$pf-p-secondary-nav__link--PaddingTop:           $global--spacer--md !default;
+$pf-p-secondary-nav__link--PaddingBottom:        $global--spacer--md !default;
 
-$pf-p-secondary-nav__link--active--Color:        $pf-global--active--Color !default;
-$pf-p-secondary-nav__link--active--BorderColor:  $pf-global--active--Color !default;
+$pf-p-secondary-nav__link--active--Color:        $global--Color--active !default;
+$pf-p-secondary-nav__link--active--BorderColor:  $global--Color--active !default;
```
Comment (cliffpyles) [inline src/patternfly/patterns/SecondaryNav/styles.scss]: I think this one was probably meant to be something else. It's using a variable that is only meant to be used with the color property.

**Comment by andresgalante** [comment]
@cliffpyles I think I've removed them all, can you take another look please?

**Pull request merged**